src/azure-cli/azure/cli/command_modules/cosmosdb/_validators.py

```diff
@@ -531,3 +531,33 @@ def _validate_pk_paths_in_cep(path, partition_key_path, policyFormatVersion, enc
                 raise InvalidArgumentValueError(f"Partition key path:{pkPath} is part of "
                                                 "Client Encryption policy with invalid encryption type. "
                                                 "Only deterministic encryption type is supported.")
+
+def validate_server_certificates(ns):
+    """ Extracts multiple comma-separated certificates """
+    if ns.server_certificates is not None:
+        ns.server_certificates = get_certificates(ns.server_certificates)
+
+
+def get_certificates(input_certificates):
+    from azure.mgmt.cosmosdb.models import Certificate
+    certificates = []
+    for item in input_certificates:
+        certificate = get_certificate(item)
+        certificates.append(Certificate(pem=certificate))
+    return certificates
+
+
+def get_certificate(cert):
+    """ Extract certificate from file or from string """
+    from azure.cli.core.util import read_file_content
+    import os
+    certificate = ''
+    if cert is not None:
+        if os.path.exists(cert):
```

Comment:
The comment for this parameter does not indicate that you can pass a string, it only mentions comma separated list of files.  This behavior of looking for the existence of a file and then changing behavior is extremely error prone, as if customer has a typo in the filename they end up with invalid certificate format which can lead to other problems.  I'm not sure the ability to pass the cert as a string is worth the problems it causes.

Reply:
made changes so that we only read from the path/to/file provided and fail if file/path does not exists.